### ISSUE TITLE
want to use mobilenetV2（inverted residual） in this model

### ISSUE THREAD

**Issue by nuptwuchen** [ildoonet/tf-pose-estimation#88]
def separable_conv(self, input, k_h, k_w, c_o, stride, name, relu=True, set_bias=True):
    with slim.arg_scope([slim.batch_norm], decay=0.999, fused=common.batchnorm_fused, is_training=self.trainable):

            input_filters = input.shape[3].value
            res_block = input
            res_block = slim.conv2d(inputs=res_block, num_outputs=input_filters * 6, kernel_size=[1, 1])
            res_block = slim.separable_conv2d(inputs=res_block, num_outputs=None, kernel_size=[3, 3], stride=stride)
            res_block = slim.conv2d(inputs=res_block, num_outputs=c_o, kernel_size=[1, 1], activation_fn=None)
        
            if stride == 2:
                return res_block
            else:
                if input_filters != output_filters:
                    net = slim.conv2d(inputs=net, num_outputs=c_o, kernel_size=[1, 1], activation_fn=None)
                return tf.add(res_block, net)

However, ..........

**Comment by ouceduxzk** [comment]
however.... there is no pretrained model, hahaha

**Comment by nuptwuchen** [comment]
@ouceduxzk 
hahaha，yes 
and I can not start run.py even if I change nothing.
If u can help me see the under question. hahaha

**Comment by ouceduxzk** [comment]
I tried with mobilenet-v2 and training from scratch, I did not tune hyperparameters much, the loss current does not going down so far, hanging around 500

**Comment by ouceduxzk** [comment]
@1icas  yes, the loss is a little higher comparing to the mobilenet-v1, you can check my fork

**Comment by sijun-zhou** [comment]
@nuptwuchen @ouceduxzk 
Hi, 
Did you finally success in training tf-pose in mobilenet v2. I tried to use mobilenet v2 in @ouceduxzk 's fork to train from scratch. But the loss of paf does not go down. the paf loss does not go down since 5000 iterations
dataset : COCO
batchsize: 16
init_lr: 0.0001
i did not change any other hyper parameters.
But the the paf loss is shown as below:
![image](https://user-images.githubusercontent.com/39326729/49009903-af266c80-f1ac-11e8-9422-938a030136e8.png)

Could any of you give me some hints? Thanks in advance!
